packages/utils/src/npm.ts

```diff
@@ -127,14 +130,14 @@ export function validateNpmSnapManifest(
 
   if (packageJsonName !== manifestPackageName) {
     throw new ProgrammaticallyFixableSnapError(
-      `${errorPrefix}"${NpmSnapFileNames.Manifest}" npm package name ("${manifestPackageName}") does not match the "${NpmSnapFileNames.PackageJson}" "name" field ("${packageJsonName}").`,
```

Comment:
Why are we removing this?

Reply:
I removed `errorPrefix` because it doesn't necessarily make sense in contexts other than the CLI. Packages that implement these functions are now responsible for adding the proper prefix to the error, if applicable.